pkg/compiler/codegen.go

```diff
@@ -1970,6 +1970,46 @@ func (c *codegen) convertBuiltin(expr *ast.CallExpr) {
 		bytes := uint160.BytesBE()
 		emit.Bytes(c.prog.BinWriter, bytes)
 		c.emitConvert(stackitem.BufferT)
+	case "min":
+		for i, arg := range expr.Args {
+			typ, ok := c.typeOf(arg).(*types.Basic)
+			if !ok {
+				c.prog.Err = fmt.Errorf("only integer types are supported by min")
+				return
+			}
+			switch typ.Kind() {
+			case types.Int, types.Int8, types.Int16, types.Int32, types.Int64,
+				types.Uint, types.Uint8, types.Uint16, types.Uint32, types.Uint64,
+				types.UntypedInt:
+			default:
+				c.prog.Err = fmt.Errorf("only integer types are supported by min, "+
+					"got %s at %d", typ, i+1)
+				return
+			}
+		}
+		for range len(expr.Args) - 1 {
+			emit.Opcodes(c.prog.BinWriter, opcode.MIN)
+		}
```

Comment:
Don't we need to add `ast.Walk` over arguments prior to this block of code? The unit-test is passed, but at the same time I don't see the code that emits arguments to VM program.

Reply:
Also, add a unit-test (a single case with 3 args is enough) that uses a *function call* instead of the raw integer as an argument to `min`/`max`.

Reply:
Yes, you're right, I missed this call to `ast.Walk`, then it's OK.